.github/workflows/synchronizing.yaml

```diff
@@ -0,0 +1,62 @@
+name: Sunc integration-definitions
+
+on: 
+  push:
+    paths:
+      - "aws-integrations/lambda-integrations/**"
+jobs:
+  Crreate_pr:
+    runs-on: ubuntu-latest
+    name: Crreate pr
+    env:
+      integration_list: 's3-sns,cloudtrail,cloudwatch-logs,s3,cloudtrail-sns,vpc-flow-logs'
```

Comment:
Would it be possible to run it in all the integrations (without whitelist)? What's stopping us?